tests/unit/flows/test_graph_flow.py

```diff
@@ -0,0 +1,67 @@
+from omnibridge.flows.graph_flow import Node, GraphFlow
+from omnibridge.model_entities.models_io.base_model_io import ModelIO, TextualIO, ImageIO
+from omnibridge.wrappers.wrapper_interfaces.processor import Processor
+
+
+class MockProcessor(Processor):
+    output = "test_output"
+
+    def __init__(self, return_value: ModelIO = TextualIO(output)):
+        self.return_value = return_value
+
+    def process(self, model_input: ModelIO) -> ModelIO:
+        return self.return_value
+
+
+def test_single_node_graph_flow():
+    # Arrange
+    node = Node(MockProcessor())
+    flow = GraphFlow([node])
+    flow_input = TextualIO("test_input")
+
+    # Act
+    flow_output = flow.process(flow_input)
+
+    # Assert
+    assert flow_output.get_inputs_aggregated_amount() == 1
```

Comment:
I noticed that `get_inputs_aggregated_amount()` is only used for tests - does it have more use cases?
If not, we can achieve something similar by mocking `Node.process()`  and then assert how many times it was called.
wdyt?